cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -185,7 +185,7 @@ public void execute() {
                 Path findRoot = ProjectDirs.findProjectRoot(this.sourceRootPath);
                 if (null == findRoot) {
                     CommandUtil.printError(this.errStream,
-                            "you are trying to build/compile a ballerina project but there is no Ballerina.toml file.",
+                            "There is no Ballerina.toml file to build/compile a Ballerina project.",
```

Comment:
```suggestion
                            "A Ballerina.toml file is required to compile/build a Ballerina project.",
```

Reply:
Changed to the following:

"need a Ballerina.toml file to build/compile a Ballerina project."